test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -116,5 +135,26 @@ private void SetPatientInfo(Patient patient, string city, string family, string
 
             patient.Name = new List<HumanName> { new HumanName { Family = family }, };
         }
+
+        private async Task<Observation[]> CreateObservations(string tag)
```

Comment:
Could you change the name of the CreateResources and CreatePaginatedResources methods to show that they make Patient resources?